.travis.yml

```diff
@@ -1,12 +1,15 @@
 language: node_js
```

Comment:
not sure why is this file here? Is this for your own CI cycle?

Reply:
It takes care of running tests and passing the build or failing in case there are any errors due to code changes